scan.py

```diff
@@ -228,6 +247,10 @@ def lambda_handler(event, context):
         if s3_object.content_length >= ALLOWED_FILE_SIZE:
             print(f'The file {s3_object.key} is too large (> {ALLOWED_FILE_SIZE / (1024 * 1024)}): '
                   f'Skipping AV scanning')
+
+            # Set SKIP_LARGE_FILE tag to the file
+            set_tags(s3_client=s3_client, s3_object=s3_object, tags_data={SKIP_LARGE_FILE: True}, update=True)
```

Comment:
It might make more sense to do send the tag value as part of AV_STATUS_METADATA which will be stored in the Physical Object in the database.

Reply:
Yeah, but it is handled by `update` param.

Reply:
I mean that the tag KEY should be `AV_STATUS_METADATA` and the value should `SKIP_LARGE_FILE`

The file at this point should not have any tags at all, so I'm not sure why the update flag is needed.

Reply:
Understand. You want to have e.g. `large-file` tag associated with the LogicalObject for such files. 

Because we already have a webhook in API that handles object tagging, I think the simplest solution is to add the tag to the S3 file in Lambda which will trigger [process_object_tagged](https://github.com/solvebio/api.solvebio.com/blob/238b3da0155939e1b5e62012df0060ca74620fe5/api/webhooks/models.py#L461) webhook, where tags will be checked and the `large-file` tag will be added to the LogicalObject if it exists.